gateway/src/service.rs

```diff
@@ -480,10 +481,35 @@ impl GatewayService {
                     state: project,
                 })
             } else {
-                // Otherwise it already exists. Because the caller of this
-                // command is the project owner, this means that the project
-                // is already running.
-                Err(Error::from_kind(ErrorKind::ProjectAlreadyRunning))
+                // Otherwise it already exists. Because the caller of this command is the
+                // project owner, this means that the project is already in some running state.
+                let state = State::from(project);
+                let message = match state {
+                    // Ongoing processes.
+                    State::Creating { .. }
+                    | State::Attaching { .. }
+                    | State::Recreating { .. }
+                    | State::Starting { .. }
+                    | State::Restarting { .. }
+                    | State::Stopping
+                    | State::Rebooting
+                    | State::Destroying => {
+                        format!("project '{project_name}' is already {state}. You can check the status again using `cargo shuttle project status`.")
+                    }
+                    // Use different message than the default for `State::Ready`.
+                    State::Ready => {
+                        format!("project '{project_name}' is already running")
+                    }
+                    State::Started | State::Stopped | State::Destroyed => {
```

Comment:
If a project is in the `Stopped` state it is idled, so perhaps we should have a separate error for this saying that the project is idled, and perhaps link to the idle projects docs?

Reply:
Ok, sounds good. I was unsure about the purpose of the `Stopped` state, so I chose to keep the message pretty vague. Thanks for explaining its purpose.

Reply:
No worries, the naming of these states is not the best, we haven't really updated the names as we changed what they represent. This is all going away in a refactor we are working on, project management will be abstracted from the user and rolled into `deploy`, but it will be some time before that is released.